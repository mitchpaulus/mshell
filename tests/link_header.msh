'<https://api.example.com/items?page=2>; rel="next"; type="application/json"; hreflang="en"' parseLinkHeader links!
@links len wl
@links :0: keys str wl
@links :0: 'url' get? wl
'<https://api.example.com/items?page=1>; rel=prev, <https://api.example.com/items?page=5>; rel="last"; title="Final \"page"' parseLinkHeader links!
@links len wl
@links :-1: 'params' get? 'title' get? wl
