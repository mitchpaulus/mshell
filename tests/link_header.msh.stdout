1
["params" "rel" "url"]
https://api.example.com/items?page=2
2
Final "page
